pages/03.Tutorials/23.using-service-accounts-with-rbac/default.en.md

```diff
@@ -0,0 +1,118 @@
+---
+title: 'Using service accounts with rbac roles'
+published: true
+taxonomy:
+    tag:
+        - kubernetes
+        - service accounts
+        - rbac
+        - user management
+---
+
+The default MetaKube Cluster comes with one admin account to use.
```

Comment:
How do you use it? It is not clear that this one admin account is used with the kubeconfig you can download from the dashboard.

Reply:
This paragraph could also benefit of a bit more details on what to expect from this tutorial, like:

"This tutorial shows how you can give additional users access to a cluster via Service Accounts (link to k8s documentation), either with full admin privileges or limited permissions."